dashboard/cron.py

```diff
@@ -80,6 +80,37 @@ class DashboardCronJob(CronJobBase):
     schedule = Schedule(run_at_times=settings.RUN_AT_TIMES)
     code = 'dashboard.DashboardCronJob'    # a unique code
 
+    # return a list of invalid course ids
+    def invalid_course_ids(self):
+
+        # whether all course ids are valid ids
+        invalid_course_id_list = []
+
+        logger.debug("in checking course")
+
+        # loop through multiple course ids
+        for course_id in Course.objects.get_supported_courses():
+            if (not course_id.isdigit()):
+                # course id can only have digit character inside
+                logger.error(f"""Course {course_id} is invalid. """)
+                invalid_course_id_list.append(course_id)
+            else:
+                # select course based on course id
+                course_sql = f"""select * 
+                            from course_dim c
+                            where c.id = '{course_id}'
+                            """
+                logger.debug(course_sql)
+                course_df = pd.read_sql(course_sql, conns['UDW'])
+
+                # error out when course id is invalid
+                if course_df.empty:
+                    logger.error(f"""Course {course_id} is invalid. """)
```

Comment:
I don't think it is an invalid course_id but there is no info in the UDW for that course. So that msg should is appropriate to that `f"Course {course_id} don't have the entry in UDW yet"`